web/router.ex

```diff
@@ -25,7 +25,7 @@ defmodule Streamr.Router do
       resources "/streams", StreamController, only: [:index]
     end
 
-    resources "/streams", StreamController, only: [:index, :create, :show] do
+    resources "/streams", StreamController, only: [:index, :create, :show, :update, :delete] do
```

Comment:
Since we have all of our 5 default actions, we can remove the `only: [...]` and just have `resources "/streams", StreamController do`